components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -143,8 +178,9 @@ - (void)testSnackbarSetAccessibiltyViewIsModalForActionSnacbars {
   XCTAssertTrue(self.manager.internalManager.overlayView.accessibilityViewIsModal);
 }
 
-- (void)testSnackbarAccessibiltyViewIsModalShouldBeNoWithNoActions {
+- (void)testSnackbarAccessibiltyViewIsModalYesWithNoActions {
   // Given
+  self.manager.internalManager.isVoiceOverRunningOverride = YES;
```

Comment:
Why is the override required now for the test? This looks like a regression/change in expected behavior.